java-lib/src/main/java/org/flinkextended/clink/feature/onehotencoder/ClinkOneHotEncoderModel.java

```diff
@@ -171,7 +171,9 @@ public Row map(Row row) throws IOException {
         @Override
         public void close() throws Exception {
             super.close();
-            ClinkJna.INSTANCE.OneHotEncoderModel_delete(modelPointer);
+            if (modelPointer != null) {
```

Comment:
It is not clear from the doc whether this close() can be called multiple times. To be extra safe, could we also set `modelPointer = null` here?